packages/odyssey-storybook/src/components/odyssey-mui/Toast/Toast.stories.tsx

```diff
@@ -267,7 +251,45 @@ export const Dismissible: StoryObj<ToastProps> = {
     linkUrl: "#",
   },
   play: async ({ args, canvasElement, step }) => {
-    openToast({ canvasElement, step })(args, "Dismissible Toast");
+    const canvas = within(canvasElement);
+    await step(`open Dismissible Toast}`, async () => {
+      await waitFor(() => {
+        const buttonElement = canvas.getByText(`Open ${args.severity} toast`);
+        userEvent.hover(buttonElement);
+        userEvent.click(buttonElement);
+        userEvent.tab();
+      });
+    });
+
+    await step("link in toast", async () => {
+      await waitFor(() => {
+        const toastLink = canvas.getByText(args.linkText || "");
+        expect(toastLink).toHaveAttribute("href", args.linkUrl);
+      });
+    });
+
+    await step("dismiss toast and reopen", async () => {
+      await waitFor(() => {
+        const toastElement = canvas.getByRole("status");
+        if (toastElement) {
+          const dismissToastButton = toastElement.querySelector(
+            '[aria-label="close"]',
+          );
```

Comment:
I know this was existing code, but why don't we just do `toastElement.getByRole("button", { name: "close" })`? There's a way to do that with Testing Library: making a selection then finding children in that selection.

Reply:
`toastElement` is HTMLElement so `getByRole` does not work. I can switch to canvas instead.
```
const dismissToastButton = canvas.getByRole("button", { name: "close" });
```

Reply:
I remember now! It's `within` that we should use. This converts a DOM element into a Testing Library selector.
```ts
within(toastElement).getByRole("button", { name: "close" })
```
Is that good?